api/bazel/repositories.bzl

```diff
@@ -21,14 +21,62 @@ def api_dependencies():
 load("@com_google_protobuf//:protobuf.bzl", "cc_proto_library", "py_proto_library")
 load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
 
+filegroup(
```

Comment:
If we keep doing this for every Google API proto we bring in, it will be quite verbose. I think it's outside the scope of this PR, but we should consider writing a Skylark macro to shrink the boilerplate.

Reply:
I added a TODO for this. I hope that's OK for now.